projects/noloan/app/src/main/java/hasadna/noloan/firestore/FirestoreClient.java

```diff
@@ -16,45 +18,54 @@
 import hasadna.noloan.protobuf.SmsProto.SmsMessage;
 
 public class FirestoreClient {
-  public static final String SPAM_COLLECTION_PATH = "noloan/spam/sms";
+  public static final String MESSAGES_COLLECTION_PATH = "noloan/spam/sms";
 
-  public static final String USER_SUGGEST_COLLECTION = "noloan/spam/suggestion";
+  public static final String MESSAGES_COLLEdCTION_PATH = "noloan/spam/suggestion";
```

Comment:
Looks like this is not used

Reply:
Yea... I had some trouble "letting go" :)

Reply:
LOL